zrml/prediction-markets/src/lib.rs

```diff
@@ -1981,10 +1973,19 @@ mod pallet {
                                 err
                             );
                         }
+                    } else if let Err(err) = pool_id {
+                        log::error!(
+                            "[PredictionMarkets] Cannot find pool associated to market.
+                            market_id: {:?}, error: {:?}",
+                            subsidy_info.market_id,
```

Comment:
Good that you use the market_id now instead of the [pool result here](https://github.com/zeitgeistpm/zeitgeist/blob/9b1e7091bf429d87a590a74de6f6941d5c50e70f/zrml/prediction-markets/src/lib.rs#L1813).